Assets/SEE/Game/MoveGizmo.cs

```diff
@@ -0,0 +1,109 @@
+﻿using UnityEngine;
+
+namespace SEE.UI3D
+{
+    /// <summary>
+    /// This gizmo represents the movement of elements of a city visually.
+    /// </summary>
+    internal class MoveGizmo : MonoBehaviour
+    {
+        /// <summary>
+        /// The start position of the movement visualization.
+        /// </summary>
+        private Vector3 start;
+
+        /// <summary>
+        /// The end position of the movement visualization.
+        /// </summary>
+        private Vector3 end;
+
+        /// <summary>
+        /// The material of the filled rectangle.
+        /// </summary>
+        private Material fillRectangleMaterial;
+
+        /// <summary>
+        /// The material for the outlined rectangle.
+        /// </summary>
+        private Material outlineRectangleMaterial;
+
+        /// <summary>
+        /// The material of the line between the start- and end-position.
+        /// </summary>
+        private Material directLineMaterial;
+
+        private static readonly int colorProperty = Shader.PropertyToID("_Color");
```

Comment:
Missing documentation. (Unless we don't want to document cached shader properties like these—there are some other instances of this in `RotateGizmo.cs` too.)

Reply:
Fixed. Since I have not written this code, but just separated it from a large incohesive file, I can sometimes just guess what those shader properties are used for.